src/tree.rs

```diff
@@ -335,6 +337,9 @@ impl Tree {
     /// assert_eq!(&processed.get(b"k3").unwrap().unwrap(), b"yappin' ligers");
     /// # Ok(()) }
     /// ```
+    ///
+    /// Note simple string prepeding in the example above should be natively changed
+    /// to some complex operation, as transactions may be tried multiple times.
```

Comment:
incorrect. transactions are serializable.

Reply:
Shall I try to construct some concrete example (involving external I/O from within a transaction) where this break down?